apc/proxy.php

```diff
@@ -9,12 +9,35 @@
 	}
 }
 
+/**
+ * Determines if SSL is used.
+ *
+ * @see is_ssl() (wp-includes/load.php).
+ *
+ * @return bool True if SSL, otherwise false.
+ */
+function cachify_is_ssl() {
+	if ( isset( $_SERVER['HTTPS'] ) ) {
+		if ( 'on' == strtolower( $_SERVER['HTTPS'] ) ) {
```

Comment:
I'm not sure if that's what we want here. I'm 100% with you in general, but ...

1. The method is copy&paste from the original WP method (used to generate the hash). I intentionally did not change the code for consistency with both calls
2. I really does not make any difference here, as `strtolower()` always returns a string.

If you still want this, I'll change it, but I don't see any benefit here and the reason is stated in the PHPDoc...